README.md

```diff
@@ -1,35 +1,28 @@
-![Seneca Tangocard-Provider](http://senecajs.org/files/assets/seneca-logo.png)
+![Seneca Webflow-Provider](http://senecajs.org/files/assets/seneca-logo.png)
```

Comment:
check with seneca-maintain

Reply:
91ff016

Reply:
exist_dist failing when it shouldn't - issue logged on @seneca/maintain